nav_msgs/Path path
---
std_msgs/Float64MultiArray  params
